persistence-cassandra-3.11/src/main/java/io/stargate/db/cassandra/impl/CassandraPersistence.java

```diff
@@ -309,6 +310,25 @@ public boolean isInSchemaAgreement() {
         <= 1;
   }
 
+  @Override
+  public boolean isInSchemaAgreementWithStorage() {
```

Comment:
No extra tests for these new `isInSchemaAgreementWithStorage` implementations?

Reply:
+1 on adding tests

Reply:
+1 on adding test(s)

Reply:
tests added